TSC-Charter.md

```diff
@@ -74,9 +74,25 @@ including:
 * Development process and any coding standards.
 * Mediating technical conflicts between Collaborators or Foundation
   projects.
+* Official communication about the above and what they govern, and the development process of said communication.
 
 The TSC will define Node.js project’s release vehicles.
 
+Specific exclusions from the communication oversight responsibilities include:
```

Comment:
This should address James' comment in https://github.com/nodejs/TSC/pull/1754#discussion_r2171825790
```suggestion
The communication oversight responsibilities include:

* Technical development processes of the project
* Releases and release announcements
* Technical documentation
* Contributor onboarding and conduct
* Project governance
* Technical direction

Specific exclusions from the communication oversight responsibilities include:
```